<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QT_IRRClass</class>
 <widget class="QMainWindow" name="QT_IRRClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>858</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QT_IRR</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QIrrlichtWidget" name="irrWidget" native="true">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>651</width>
      <height>631</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="maya_cam">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>f</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fps_cam">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>f</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>924</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QIrrlichtWidget</class>
   <extends>QWidget</extends>
   <header>QIrrlichtWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QT_IRR.qrc"/>
 </resources>
 <connections/>
</ui>
